snapshots/btrfs/btrfs_test.go

```diff
@@ -29,7 +29,13 @@ func boltSnapshotter(t *testing.T) func(context.Context, string) (snapshots.Snap
 
 	return func(ctx context.Context, root string) (snapshots.Snapshotter, func() error, error) {
 
-		deviceName, cleanupDevice, err := testutil.NewLoopback(650 << 20) // 650 MB
+		loopbackSize := int64(100 << 20) // 100 MB
+		// mkfs.btrfs needs > 650MB space on ppc64le because of its larger default pagesize (64KB vs 4KB)
```

Comment:
`sysconf(_SC_PAGESIZE)`?

Reply:
@AkihiroSuda I'm not sure what you are asking. That is one of the ways to get the value, yes.

Reply:
I meant we can check pagesize rather than GOARCH.

Reply:
ahh yeah that would be probably be better